### PR TITLE
feat(virtualization): serial/vnc reconnect

### PR DESCRIPTION
**Pull request by danilrwx**
Expected console operation:
- Reconnect every second if connection is lost (network, reboot, VM shutdown)
- If someone else is connected or VM is not found, do not try to reconnect.

### CODE REVIEW

**Review by yaroslavborbat — APPROVED** [review]

File: internal/virtualization/cmd/console/console.go [hunk 1]
```diff
@@ -86,6 +87,64 @@ func (c *Console) Run(args []string) error {
...
+	}()
+
+	for {
+		err := connect(name, namespace, virtCli, stdinCh)
+		if err != nil {
+			if errors.Is(err, util.ErrorInterrupt) || strings.Contains(err.Error(), "not found") {
+				return nil
+			}
+
+			if e, ok := err.(*websocket.CloseError); ok && e.Code == websocket.CloseGoingAway {
```
Comment (mvasl) [inline internal/virtualization/cmd/console/console.go:127]: Use of https://pkg.go.dev/errors#As is considered the best practice for such cases. You can also check error codes after matching error type wiat plain `switch`, which will be more readable

File: internal/virtualization/cmd/vnc/vnc.go [hunk 2]
```diff
@@ -134,6 +134,52 @@ func (o *VNC) Run(cmd *cobra.Command, args []string) error {
...
 	// the goroutines and let the data flow
 
+	for {
+		err := connect(ln, virtCli, cmd, namespace, vmName)
+		if err != nil {
+			if strings.Contains(err.Error(), "not found") {
+				return err
+			}
+
+			if e, ok := err.(*websocket.CloseError); ok && e.Code == websocket.CloseGoingAway {
```
Comment (mvasl) [inline internal/virtualization/cmd/vnc/vnc.go:144]: Same as above

**Review by mvasl — CHANGES_REQUESTED** [review]

File: internal/virtualization/cmd/vnc/vnc.go (hunk 2 above)
Comment (danilrwx) [reply internal/virtualization/cmd/vnc/vnc.go]: fixed

File: internal/virtualization/cmd/console/console.go (hunk 1 above)
Comment (danilrwx) [reply internal/virtualization/cmd/console/console.go]: fixed

**Review by mvasl — APPROVED** [review]

**Pull request merged**